README.md

```diff
@@ -24,8 +24,24 @@ There are no dependencies.
 
 ## Usage
 
-Below is a sample playbook:
+Below are sample usages for playbook:
 
+##### For deploying agents to Non-Multi-Tenancy Deep Security Manager
+```yaml
+- hosts: all
+  roles:
+    - role: deep-security.deep-security-agent
+      action: deploy
+      dsm_agent_download_hostname: deep.security.manager.host
+      dsm_agent_download_port: 4119
+      dsm_agent_activation_hostname: deep.security.manager.host
+      dsm_agent_activation_port: 4120
+      policy_id: 1
+      group_id: 1
+      force_reactivation: false
+```
+
+##### For deploying agents to Multi-Tenancy Deep Security Manager (like Deep Security as a Service),
```

Comment:
For deploying [...]-> For deploying agents to a multi-tenant Deep Security Manager (like Deep Security as a Service)